go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/gophercloud/utils v0.0.0-20200204043447-9864b6f1f12f
 	github.com/onsi/ginkgo/v2 v2.1.4
 	github.com/onsi/gomega v1.20.0
+	github.com/pkg/errors v0.9.1
```

Comment:
Why you now introduce this when there was explicit PR to clean up this dependency? Ref https://github.com/gardener/gardener-extension-provider-openstack/pull/331

Reply:
Yes, we should remove `github.com/pkg/errors` again. (Easy to happen for long running PRs like this).